src/scales.js

```diff
@@ -471,6 +471,384 @@ export function coerceDate(x) {
     : new Date(x);
 }
 
+/**
+ * You can also create a standalone scale with Plot.**scale**(*options*). The *options* object must define at least one scale; see [Scale options](https://github.com/observablehq/plot/blob/main/README.md#scale-options) for how to define a scale. For example, here is a linear color scale with the default domain of [0, 1] and default scheme *turbo*:
+ *
+ * ```js
+ * const color = Plot.scale({color: {type: "linear"}});
+ * ```
+ *
+ * #### Scale objects
+ *
+ * Both [*plot*.scale](https://github.com/observablehq/plot/blob/main/README.md#plotscalescalename) and [Plot.scale](https://github.com/observablehq/plot/blob/main/README.md#plotscaleoptions) return scale objects. These objects represent the actual (or “materialized”) scale options used by Plot, including the domain, range, interpolate function, *etc.* The scale’s label, if any, is also returned; however, note that other axis properties are not currently exposed. Point and band scales also expose their materialized bandwidth and step.
+ *
+ * To reuse a scale across plots, pass the corresponding scale object into another plot specification:
+ *
+ * ```js
+ * const plot1 = Plot.plot(…);
+ * const plot2 = Plot.plot({…, color: plot1.scale("color")});
+ * ```
+ *
+ * For convenience, scale objects expose a *scale*.**apply**(*input*) method which returns the scale’s output for the given *input* value. When applicable, scale objects also expose a *scale*.**invert**(*output*) method which returns the corresponding input value from the scale’s domain for the given *output* value.
+ *
```

Comment:
Maybe we could stop here for the jsdoc, and leave the rest in README. (The position options etc are mostly meant in the context of writing scale descriptions as top-level options of the plot.)

Reply:
that makes sense, I'll make that change today

Reply:
done!